src/Presenters/Content/WikiContentProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Presenters\Content;
```

Comment:
I actually wanted to remove/edit my comment but my phones connection failed.

Currently the code in the Presenters folder only has presenters that all only depend on response model objects, value objects from the domain and presentation services (ie the TwigTemple). There are no dependencies on domain services or anything data access related. That was my original idea of this folder, just such presenters, and perhaps parts of them if some get split into multiple classes.

Perhaps we need a more general `Presentation` folder, in which `Presenters` is a sub folder? This can then hold presentation services and whatever `WikiContentProvider` and `PageContentModifier` are.